unfuck-submodules.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIRNAME=$(basename "$PWD")
+if [[ "$DIRNAME" != "hyperlane-monorepo" ]]; then
+  echo "Must be run from the root of the monorepo"
+  exit 1
+fi
+
+DIRS=$(git submodule status | grep -oE "typescript/[a-zA-Z0-9_-]+")
+for DIR in $DIRS ; do
+    echo "Removing '$DIR'"
+    rm -rf "$DIR"
+done
```

Comment:
So this removes any dirs that have a TS change?

Reply:
It removes any submodules that are in the typescript directory.